droptools-example/src/main/java/com/bendb/example/ExampleApp.java

```diff
@@ -32,13 +34,26 @@ public FlywayFactory getFlywayFactory(ExampleConfig configuration) {
         bootstrap.addBundle(new JooqBundle<ExampleConfig>() {
             @Override
             public DataSourceFactory getDataSourceFactory(ExampleConfig configuration) {
-                return configuration.dataSourceFactory();
+                return null;
```

Comment:
This whole method is ignored if getDataSourceFactories() is overridden. I've added comments in places to help clarify this.